Does not build on windows

<!--
  ⚡️ katchow! We 💛 issues.

  🚨 Your issue will be CLOSED if:
     - This template is removed
     - Parts of this template are removed

  👉🏽 Need help or tech support? Please don't open an issue!
  Head to https://discord.gg/FywZN57mTg or start a Discussion https://github.com/shellscape/jsx-email/discussions

  ❤️ JSX email? Please consider sponsoring our maintainers or telling others about us!
-->

- Component or Package Name: cli
- Component or Package Version: 1.1.0
- `@jsxp-email/cli` Version?: 1.1.0
- Operating System (or Browser): Windows 11
- Node Version: 18.14.2
- Link to reproduction _(⚠️ read below)_: Brand new project on Windows, created with `npm create jsx-email`, no need for reproduction.

<!--
  🚨 Issues WITHOUT a valid reproduction WILL BE CLOSED!

  Please provide one by:
  1. Using the StackBlitz reproduction template at https://stackblitz.com/fork/jsx-email-repro
  2. Provide a minimal repository link (Read https://gist.github.com/Rich-Harris/88c5fc2ac6dc941b22e7996af05d70ff for instructions).
     Please use NPM for installing dependencies!
     These may take more time to triage than the other options.

  ⚠️ ZIP Files are unsafe and maintainers will NOT download them.
-->

### Expected Behavior

Build works on Windows

### Actual Behavior

It doesn't

### Additional Information

Can't create PR now
Build command uses `globby`, which returns empty arrays on windows paths. You should use `globby([slash(join(temp, "*.js")])` everywhere in CLI.
Thanks

Oh my bad. The issue is because I did not name my folder `jsx-email` (whereas Moon tried to decipher the project name from folder name). I am now using WSL (since the command `rm -rf` does not work on Windows), and ran into a new error after running `moon repo:build.packages`

```bash
src/tailwind.tsx(2,29): error TS2307: Cannot find module '@jsx-email/render' or its corresponding type declarations
```

--- a/packages/cli/src/commands/build.ts
+++ b/packages/cli/src/commands/build.ts
@@ -1,7 +1,7 @@
 import { existsSync } from 'node:fs';
 import { mkdir, realpath, writeFile } from 'node:fs/promises';
 import os from 'node:os';
-import { basename, extname, join, resolve } from 'path';
+import { basename, extname, join, resolve, win32, posix } from 'path';
 
 import { render } from '@jsx-email/render';
 import chalk from 'chalk';
@@ -61,6 +61,9 @@ const pretty = (html: string) => {
   return beautify.html(html, defaults);
 };
 
+// Credit: https://github.com/rollup/plugins/blob/master/packages/pluginutils/src/normalizePath.ts#L5
+const normalizePath = (filename: string) => filename.split(win32.sep).join(posix.sep);
+
 const stripHtml = (html: string) => {
   const $ = load(html);
 
@@ -108,7 +111,7 @@ const compile = async (files: string[], outDir: string) => {
     write: true
   });
 
-  return globby([join(outDir, '*.js')]);
+  return globby([normalizePath(join(outDir, '*.js'))]);
 };
 
 export const command: CommandFn = async (argv: BuildOptions, input) => {
@@ -129,7 +132,7 @@ export const command: CommandFn = async (argv: BuildOptions, input) => {
   const isFile = target.endsWith('.tsx') || target.endsWith('.jsx');
   const { out = '.rendered' } = argv;
   const glob = isFile ? target : join(target, '*.{jsx,tsx}');
-  const targetFiles = await globby([glob]);
+  const targetFiles = await globby([normalizePath(glob)]);
   const outputPath = resolve(out);
 
   log('Found', targetFiles.length, 'files:');

